source/mainnet/net/voting/voting.rst

```diff
@@ -52,20 +54,20 @@ Since the votes are weighted by the number of CCD on the account, for a user to
     #. Disconnect and reconnect in the voting dApp. The user will now be connected to the account that is open in the wallet.
     #. Repeat the two last steps above to vote with all accounts.
 
-For Desktop wallet and |mw-gen1| wallet holders
-===============================================
+For Desktop wallet, |mw-gen1| and Concordium Client wallet holders
+==================================================================
 
 These older wallets do not have the capability to connect to dApps. But you do have an option to exercise your vote.
 
 .. dropdown:: Desktop wallet
 
-    Users need to create a new account in |bw|, |cryptox|, or |mw-gen2| and delegate their vote from their Desktop wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw|, |cryptox|, or |mw-gen2| at any time before the election.
+    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Desktop wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before delegating your vote.
 
     **To delegate your vote:**
```

Comment:
The way I read it, it is not explicitly said that delegation transaction must be sent while the voting window time is open. Otherwise, delegation will not be picked up.